lib/project_types/script/graphql/get_app_scripts.graphql

```diff
@@ -0,0 +1,6 @@
+query GetAppScripts($appKey: String!, $extensionPointName: ExtensionPointName!) {
+  appScripts(appKeys: [$appKey], extensionPointName: $extensionPointName) {
```

Comment:
This is a temporary issue until the rearchitecture is done, but the `RestrictedVisibility` module disallows the `appScripts` query from being made via the proxy since a partner could theoretically request any app key including their script executions.

A couple solutions:
1. Allow the `appScripts` query and disallow the `executions` fields for `partners_proxy`.
2. Expose a new query for `partners_proxy` which returns all App Scripts for the current app key.

Reply:
Spoke offline and agreed on the first approach given that this should be a temporary solution. PR is [here](https://github.com/Shopify/script-service/pull/3000).